### DISCUSSION TITLE
How to use arbitrary color value in php echo ?

### DISCUSSION THREAD

**Discussion by lingjiacode** [tailwindlabs/tailwindcss#8359] [Help]
"#1FFFC6" and "#4C59FF" are custom colors picked from some color picker in wordpress backend.

if i write the code below directly, it works just fine.
`<h1 class="bg-[#1FFFC6] text-[#4C59FF]">h1 title</h1>`

but,  this one not working.

`$nav_bgcolor = "#1FFFC6";`
`$nav_textcolor = "#4C59FF";`
`echo "<h1 class='bg-[$nav_bgcolor] text-[$nav_textcolor]'>title</h1>";`

is there any way i can do this? 
please help, thanks.

env :
php7.4, tailwind v3, vscode.

**Comment by ricardo21ferreira** [comment]
In your file: tailwind.config.js you need to add mode: 'jit'.

source: https://v2.tailwindcss.com/docs/just-in-time-mode

e.g:
`module.exports = {
   mode: 'jit',
    content: [
        './*.php',
        ......
    ],
        //other options
}`

**Comment by zslabs** [comment]
You can't dynamically create classes like that in Tailwind CSS. You'd be better off having your PHP update a CSS variable and create a utility class that references it instead. That way, you can still have a dynamic property and allow the class to compile correctly.
